databaseQueries/programUsers.js

```diff
@@ -0,0 +1,45 @@
+module.exports= class Programs{
+    static findProgramJoined(programId,userId) {
+        return new Promise(async (resolve, reject) => {
+            try {
+               let queryObject = {
+                programId :programId,
+                userId: userId
+               }
+               
+               
+               let programJoinedData = 
+               await database.models.programUsers.find(
+                   queryObject, 
+                   {}
+               ).lean();
+               return resolve(programJoinedData);
+           
+           } catch (error) {
+               return reject(error);
+           }
+       });
+   }
+
+
+   static findProgramIds(userId) {
```

Comment:
@ankitpws I think we can remove this function if we can generalise the above function. Basically both are querying same collection only query and projection is changing. please check projectsDocument function in
https://github.com/shikshalokam/ml-projects-service/blob/master/databaseQueries/projects.js